component/azstorage/block_blob.go

```diff
@@ -468,6 +468,8 @@ func (bb *BlockBlob) getAttrUsingRest(name string) (attr *internal.ObjAttr, err
 func (bb *BlockBlob) getAttrUsingList(name string) (attr *internal.ObjAttr, err error) {
 	log.Trace("BlockBlob::getAttrUsingList : name %s", name)
 
+	listPath := filepath.Join(bb.Config.prefixPath, name)
```

Comment:
Can we add a test that validates this fix?

Reply:
As of today we do not have any test case which validates subdirectory mount.

Reply:
subdirectory mount can only be validated in E2E and for those test suits we create new container and mount it so subdirectory mount can not be tested as such.

Reply:
Done, have added UT to validate the function which is suppose to remove prefix path from the received path.
Also added subdirectory based getAttr method in existing UT.